ffi/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@rules_go//go:def.bzl", "go_library")
```

Comment:
As discussed offline, `@rules_go` does seems to cause some naming conflicts / incompatibility within a repo that has `@io_bazel_rules_go` imported. Can we rename this to `@io_bazel_rules_go`?